up.go

```diff
@@ -21,6 +21,7 @@ import (
 	"bufio"
 	"bytes"
 	"context"
+	"flag"
```

Comment:
Could you please use [github.com/spf13/pflag](https://godoc.org/github.com/spf13/pflag) instead? I'm already starting to think of more flags in near future, and I think the POSIX approach will be more friendly to people in the long run.

Reply:
I agree, we should stick with pflag